index.html

```diff
@@ -69,8 +73,124 @@
 
     <section class='informative'>
       <h2>Introduction</h2>
-      <p>This document is a stub.</p>
+      <p>The Browser Extensions Community Group's goal is to make browser extension code much more interoperable across browsers by specifying common extension interfaces and well-defined browser behavior. </p>
+      <p>This specification defines the interfaces and behavior which enables extensions to communicate with local client native processes outside of the browser environment.</p>
+    </section>
+
+    <section class='informative'>
+        <h2>Core APIs</h2>
+        <p>The goal for these APIs is to provide functionality that extension authors need for:</p>
+        <ul>
+            <li>message-passing between extensions and local client processes outside of the browser</li>
```

Comment:
I replaced the list with your suggested text. I used "local client native processes" throughout. If we decide to change it, it should be an easy search-and-replace.